backend/src/database/queries/setUser.ts

```diff
@@ -0,0 +1,18 @@
+import { parsePrefixesToQuery } from '../../common/database';
+import { PREFIXES } from '../index';
+
+export default (username: string, password: string, role: string): string => {
+  const prefixString = parsePrefixesToQuery(PREFIXES.SDG);
+  return `
+    ${prefixString}
+    insert {
+      ?uri rdf:type SDG:User .
+      ?uri SDG:username "${username}".
+      ?uri SDG:passwordHash "${password}".
+      ?uri SDG:userHasRole ?role.
+   }
+   where {
+    BIND(IRI("http://www.semanticweb.org/aga/ontologies/2017/9/SDG#datapoint.u4ssc.${username}") as ?uri)
```

Comment:
I think that a different IRI-path would be more suitable, eg. `"http://www.semanticweb.org/aga/ontologies/2017/9/SDG#user.${username}"`.

Reply:
Haha ofc, copied from another query and forgot to replace 😓